omero/sysadmins/unix/server-linux-walkthrough.txt

```diff
@@ -134,11 +182,13 @@ Copy this script into the appropriate location:
 | :download:`step08_all_cron.sh <walkthrough/step08_all_cron.sh>`
 
 
+.. _SELinux:
+
 SELinux
 -------
 
 If you are running a system with SELinux enabled (for example, it is
-`enabled by default on CentOS6 <http://wiki.centos.org/HowTos/SELinux>`_)
+`enabled by default on CentOS 6 <http://wiki.centos.org/HowTos/SELinux>`_)
 and are unable to access OMERO.web you may need to adjust the security policy:
 
 .. literalinclude:: walkthrough/setup_centos6_selinux.sh
```

Comment:
As mentioned in https://github.com/openmicroscopy/ome-documentation/pull/1363#issuecomment-16467943, this should be renamed to `setup_centos_selinux.sh`